autosub/audioProcessing.py

```diff
@@ -3,11 +3,13 @@
 
 import sys
 import shlex
-from . import logger
```

Comment:
this causes a regression of #52 

also blame https://github.com/KyleMaas/AutoSub/commit/5dc2314dea2f7ffc86e1454cb2ef29c6721d4e55

please keep all the local imports from #54